mentorship ios/Views/Home/Home.swift

```diff
@@ -71,7 +82,7 @@ struct Home: View {
             }
             .listStyle(GroupedListStyle())
             .environment(\.horizontalSizeClass, .regular)
-            .navigationBarTitle("Welcome \(self.homeViewModel.userName?.capitalized ?? "")!")
+            .navigationBarTitle("Welcome \(userFirstName)!", displayMode: .inline)
```

Comment:
I don't think we want to change the display mode to inline...

Reply:
Gotcha, thx

Reply:
I was worried about super long first names